coderd/database/migrations/000063_resource_type_group.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TYPE resource_type ADD VALUE 'group'; 
```

Comment:
Since this isn't downgradable, I think a `IF NOT EXISTS` might be helpful in case it needs to be ran twice.